frameworks/hdfs/src/main/dist/ssl-client.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+    <property>
+        <name>ssl.client.truststore.password</name>
+        <value>changeit</value>
+    </property>
+    <property>
+        <name>ssl.client.truststore.type</name>
+        <value>jks</value>
```

Comment:
I think JKS is what everyone one uses in Java world. So, this might not be something that can be templated. Although, I might be wrong.